public/api/docs/v3/swagger.yaml

```diff
@@ -1093,6 +1093,7 @@ components:
         training_status:
           description: Return only records that have this training status
           type: string
+          nullable: true
```

Comment:
I don't think as far as the API is concerned this is a nullable field, it will always be populated in the participant profile response so shouldn't be nullable there